src/types/parity_pending_transaction.rs

```diff
@@ -0,0 +1,135 @@
+use serde::{
+    ser::{SerializeMap, Serializer},
+    Serialize,
+};
+
+use super::{Address, U256, U64};
+
+/// Condition to filter pending transactions
+#[derive(Clone)]
+pub enum FilterCondition<T> {
+    /// Lower Than
+    Lt(T),
+    /// Equal
+    Eq(T),
+    /// Greater Than
+    Gt(T),
+}
+
+/// To Filter
+#[derive(Clone)]
+pub enum ToFilter {
+    /// Address
+    Address(Address),
+    /// Action (i.e. contract creation)
+    Action,
+}
+
+/// Filter for pending transactions (only openethereum/Parity)
+#[derive(Clone, Default, Serialize)]
+pub struct ParityPendingTransactionFilter {
```

Comment:
Could you please add `::builder()` function to make it easy to create builder struct without importing the type?
```rust
impl ParityPendingTransactionFilter {
  pub fn builder() -> ParityPendingTransactionFilterBuilder {
    Default::default()
  }  
}
```